### PR TITLE
Updates the Provisioning Guide

### PR DESCRIPTION
**Pull request by IanHakes-okta**
## Description:
- **What's changed?**
Add new topic for configuring provisioning settings in the Okta integration after enabling the API connection.
Added a troubleshooting section to the Connect a new Okta integration page.
Added links to example implementations from the prepare your API page.
Minor update to overview page help section.
Minro update to make sections on the testing page into H2's so they show up in the right hand navigation.

### Resolves: 
* [OKTA-279538](https://oktainc.atlassian.net/browse/OKTA-279538)

- **Is this PR related to a Monolith release?**
No

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/prepare-api/index.md [hunk 1]
```diff
@@ -110,6 +110,13 @@ A best practice is to use a generated globally unique identifier (GUID) for this
 
 Okta user management requires that your SCIM API supports an `active` attribute for each user resource that can be set to `true` or `false` to denote a resource as "active" or "inactive".
 
+## Example implementations
+
+There are numerous examples of how to implement SCIM servers and applications. Here are a few to get you started:
+
+- [scimify](https://toolkit.okta.com/apps/scimify/) - A PHP application that supports both SCIM 1.1 and SCIM 2.0 servers with operations for /Users, /Groups and /ServiceProviderConfig endpoints
```
Comment (mrunalpatel-okta) [inline packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/prepare-api/index.md:117]: Apache SCIMple is good one but we shouldn't link scimify as reference implementation. It's code is not production quality.

Comment (mrunalpatel-okta) [reply packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/prepare-api/index.md:117]: Other alternative,
https://code.google.com/archive/p/openscim/

Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/prepare-api/index.md]: Examples updated based on conversation with Mrunal in slack.

File: packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/configure-app/index.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+* **Deactivate Users** &mdash; Automatically deactivates user accounts when they are unassigned in Okta or their Okta accounts are deactivated.
+
+    Okta can also reactivate the application account if the integration is later reassigned to a user in Okta.
+
+* **Exclude Username Updates** &mdash; Prevents your downstream application profile from overwriting the Okta user profile when using the profile push feature.
+
+* **Sync Password** &mdash; Ensures a user's application password is always the same as their Okta passwords or, alternatively, allows Okta to generate a unique password for the user. See [Synchronize passwords](https://help.okta.com/en/prod/okta_help_CSH.htm#ext-password-sync-main) .
+
+* **Profile Attribute Mappings** &mdash; Use this portion of the page to edit attributes and mappings in the Profile Editor. See  the [attributes and mappings](/docs/guides/build-provisioning-integration/attribute-mapping/) section or [Work with Okta user profiles and attributes](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Directory_Manage_Profile_Attributes) in the Okta product documentation.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/configure-app/index.md:33]: extra space after See?

File: packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/configure-app/index.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+This screen contains settings for all information that flows from your application upstream into Okta.
+
+Click **Edit** to make changes to the following sections.
+
+* **General** &mdash; Use this section to schedule imports and dictate a username format for imported users. You can also define a percentage of acceptable assignments before the [import safeguards](https://help.okta.com/en/prod/okta_help_CSH.htm#csh-eu-import-safeguard) feature is automatically triggered.
+
+    If the Okta username is overridden due to mapping from a provisioning-enabled app, the custom mapping appears in this section.
+
+* **User Creation & Matching** &mdash; Matching rules are used when importing users from all applications and directories that allow importing. Establishing matching criteria allows you to specify how an imported user should be defined as a new user or mapped to an existing Okta user.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/configure-app/index.md:45]: You start nearly all of the descriptions in this section with an active verb. There are a few that you don't. Can you start this one and the other 3 with an active verb?

File: packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/configure-app/index.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+  * **Imported user is an exact match to Okta user if** &mdash; The matching criteria that establishes whether an imported user exactly matches an existing Okta user. Choose any combination from the list of options to establish your criteria. For the new imported user to be considered an exact match, each option that you select must be true. Note that if you choose the third option, the first and second choices are disabled.
+
+  * **Allow partial matches** &mdash; Partial matching occurs when the first and last name of an imported user match those of an existing Okta user, but the username or email address do not.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/configure-app/index.md:49]: do not -> don't?

File: packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/configure-app/index.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+  * **Confirm matched users** &mdash; Select to automate the confirmation or activation of existing users. If the option isn't selected, matches are confirmed manually.
+
+  * **Confirm new users** &mdash; Select to automate the confirmation or activation of a newly imported user. If this option is selected, you can clear it during import confirmation. Note that this feature doesn't apply for users who already exist in Okta.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/configure-app/index.md:53]: Note that this feature doesn't apply **to** users who....

File: packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/configure-app/index.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+* **Profile & Lifecycle Mastering** &mdash; Use this section to allow the current application to profile master Okta users. If enabled, the application appears in the list of profile masters on the Profile mastering page.
+
+  * **Allow `<app>` to master Okta users** &mdash; Enable mastery and determine what happens when a user is deactivated or reactivated in an application. Only the highest priority profile master for that Okta user can deactivate or suspend an Okta user. To verify the highest priority profile master, review the Profile mastering page.
+
+  * **When a user is deactivated in the app** &mdash; Select **Do Nothing** to prevent activity in the application from controlling the user life cycle. This still allows profile master control of attributes and mappings. Otherwise, you can choose between deactivating or suspending the user.
+
+  * **When a user is reactivated in the app** &mdash; Decide if suspended or deactivated Okta users should be reactivated when they have been reactivated in the application.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/configure-app/index.md:61]: I don't understand this sentence: So, a suspended or deactivated Okta user can be reactivated and **then** the admin has to decide if they should be reactivated?

File: packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/configure-app/index.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+    >**Note** When a user is reactivated in the app, the user profile must be an exact match to the Okta profile for the reactivation to also occur in Okta. Otherwise, after importing the reactivated users, they appear in **Pending Activation** state.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/configure-app/index.md:63]: is the word also necessary here? I keep tripping over it.

File: packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/configure-app/index.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+
+* **Import Safeguards** &mdash; The import safeguard settings define the maximum percentage of application users in an org that can be unassigned while still allowing the import to proceed. App-level and org-level safeguards are enabled by default and set at 20 percent.
+
+* **Inline Hooks** &mdash; Use this section to add custom logic to the process of importing new users into Okta from an app. You can resolve conflicts in profile attributes and control whether imported users are treated as matches for existing users. To enable an import inline hook, see [Inline hooks](/docs/concepts/inline-hooks/).
+
+* **Okta Attribute Mappings** &mdash; Use this portion of the page to edit attributes and mappings in the Manage user profiles.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/configure-app/index.md:69]: this section of the page?

File: packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/prepare-api/index.md [hunk 9]
```diff
@@ -110,6 +110,17 @@ A best practice is to use a generated globally unique identifier (GUID) for this
 
 Okta user management requires that your SCIM API supports an `active` attribute for each user resource that can be set to `true` or `false` to denote a resource as "active" or "inactive".
 
+## Example implementations
+
+There are numerous examples of how to implement SCIM servers and applications. Here are a few to get you started:
+
+- [scimify](https://toolkit.okta.com/apps/scimify/) - A PHP application for Okta that supports both SCIM 1.1 and SCIM 2.0 servers with operations for /Users, /Groups and /ServiceProviderConfig endpoints
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/prepare-api/index.md:117]: replace dashes with &mdash;

also comma after /Groups - we use the serial comma

File: packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/prepare-api/index.md [hunk 10]
```diff
@@ -110,6 +110,17 @@ A best practice is to use a generated globally unique identifier (GUID) for this
...
+- [Apache Directory SCIMple](https://github.com/apache/directory-scimple) - Apache's Java EE implementation of the SCIM version 2.0 specification.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/prepare-api/index.md:118]: no period

File: packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/prepare-app/index.md [hunk 11]
```diff
@@ -63,4 +63,14 @@ Click **Test Connector Configuration** to confirm that Okta can connect to your
 1. Click **Save** to complete the SCIM integration setup.
 -->
 
+## Troubleshooting
+
+If you experience any difficulties when creating your SCIM integration in Okta, check out the system log information available in the Okta Admin Console.
+
+1. From the Admin Console for your developer org, go to **Applications > Applications**.
+1. Click on your Okta integration to open the integration settings page.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/prepare-app/index.md:71]: Click your Okta integration to open the integration settings page.

or

Select your Okta integration to open the integration settings page.

File: packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/prepare-app/index.md [hunk 12]
```diff
@@ -63,4 +63,14 @@ Click **Test Connector Configuration** to confirm that Okta can connect to your
...
+1. Click **View Logs** to open the System Log.
+
+The system log captures all events in your developer org for the previous 7 days. This information is invaluable to troubleshoot any connection or authentication issues between Okta and your application. See [System Log](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Reports_SysLog) in the Okta product documentation.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/prepare-app/index.md:74]: 7 -> seven days.

File: packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/configure-app/index.md (hunk 4 above)
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/configure-app/index.md:49]: changed to:
"... but not the username or email address."

File: packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/configure-app/index.md (hunk 8 above)
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/configure-app/index.md:69]: Replaced with text similar to the end of the "To Okta" section:
Edit attributes and mappings through the Profile Editor. See the <GuideLink link="../attribute-mapping/">Attributes and mappings</GuideLink> section in this guide or [Work with Okta user profiles and attributes](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Directory_Manage_Profile_Attributes) in the Okta product documentation.

File: packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/configure-app/index.md (hunk 6 above)
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/configure-app/index.md:61]: Looking at this text, it's not clear what is being reactivated where. I have changed it to:
"Determines if deactivated or suspended Okta user profiles that are assigned to your Okta integration should also be reactivated when they have been reactivated in your downstream application."

File: packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/prepare-api/index.md (hunk 9 above)
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/prepare-api/index.md:117]: Bad copy and pasting! Bad!

File: packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/configure-app/index.md (hunk 3 above)
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/configure-app/index.md:45]: I think I fixed them all

File: packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/configure-app/index.md [hunk 13]
```diff
@@ -0,0 +1,77 @@
+---
+title: Configure your Okta integration
+---
+
+Now that you have created a integration within your Okta org, you need to configure the SCIM options for that integration. These settings tell your Okta integration how to handle provisioning between the users in your downstream SCIM app and their Okta user profiles.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/configure-app/index.md:5]: created an integration

File: packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/overview/index.md [hunk 14]
```diff
@@ -10,7 +10,9 @@ While many ISVs have custom APIs for managing user accounts, this guide assumes
 
 Your Okta integration should use Single Sign-On (SSO) to initiate end-user authentication. Learn how to set up your integration with SSO in our [Build a Single Sign-On (SSO) integration](/docs/guides/build-sso-integration/) guide.
 
-If you need help, read through our [SCIM FAQ](/docs/concepts/scim/faqs/) page, post your question on the [Okta Developer Forums](https://devforum.okta.com/), or email us at <developers@okta.com>.
+If you need additional assistance, read through our [SCIM FAQ](/docs/concepts/scim/faqs/) page, post your question on the [Okta Developer Forums](https://devforum.okta.com/), or email us at <developers@okta.com>.
+
+## Submitting your integration
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/overview/index.md:15]: just noticed this gerund heading -> should be Submit your integration

**Pull request merged**